docs/visual-basic/language-reference/statements/if-then-else-statement.md

```diff
@@ -93,17 +103,19 @@ If condition Then [ statements ] [ Else [ elsestatements ] ]
   
  In the single-line syntax, you can have multiple statements executed as the result of an `If`...`Then` decision. All statements must be on the same line and be separated by colons.  
   
-## Example  
+<a name="multi-line"></a>
+ 
  The following example illustrates the use of the multiple-line syntax of the `If`...`Then`...`Else` statement.  
   
  [!code-vb[VbVbalrStatements#101](../../../visual-basic/language-reference/error-messages/codesnippet/VisualBasic/if-then-else-statement_1.vb)]  
   
-## Example  
+<a name="nested"></a>
+
  The following example contains nested `If`...`Then`...`Else` statements.  
   
  [!code-vb[VbVbalrStatements#102](../../../visual-basic/language-reference/error-messages/codesnippet/VisualBasic/if-then-else-statement_2.vb)]  
```

Comment:
It would also be good to wrap this in a Module...End Module statement, include a Main method so it's executable, and add output
The comments also aren't fully accurate. It doesn't test for 2 to 4 PM or noon to 1 PM, but from 2 to 3:59 and 12 to 12:59.